docs/documentation/platform/sso/overview.mdx

```diff
@@ -39,18 +39,30 @@ If your required identity provider is not shown in the list above, please reach
   For enhanced security, Infisical enforces PKCE (Proof Key for Code Exchange) with the OAuth 2.0-based SSO providers and OIDC. This provides additional protection against authorization code interception attacks and strengthens your authentication flow security.
 </Info>
 
+## Admin Login Portal
+
+Organization Admins can utilize the Admin Login Portal to bypass SSO enforcement in case of an emergency.
```

Comment:
I'd start with something like: In the event your SSO provider experiences down time, and you need to access Infisical, you can use ...